Components/ListSearchResults/Partials/Header/_style.scss

```diff
@@ -1,59 +1,83 @@
+$search-form-spacing: 6px;
+
 .header {
   &-title {
-    @include reset-content-margins;
-
-    padding: $container-padding-desktop 0;
+    text-align: center;
   }
 }
 
 .searchForm {
+  position: relative;
+
   &-label {
-    height: 1px;
-    overflow: hidden;
+    left: 10px;
+    margin-bottom: 0;
     position: absolute;
-    width: 1px;
+    top: 50%;
+    transform: translateY(-50%);
+    z-index: 10;
+  }
+
+  &-icon {
+    color: $color-headline;
   }
 
   &-field {
+    background-color: #fff;
+    border: 1px solid #fff;
+    border-radius: 4px;
+    box-shadow: 0 1px 26px 0 rgba(0, 0, 0, 0.06);
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    padding: $search-form-spacing;
+    position: relative;
+    width: 100%;
+
+    @media (min-width: $breakpoint-mobile-horizontal) {
+      flex-direction: row;
+    }
+  }
+
+  &-box {
+    flex-grow: 1;
+    padding-left: 50px;
+    padding-right: $search-form-spacing;
     position: relative;
   }
 
   &-input {
-    background-color: #fff;
-    border: 2px solid $color-accent;
+    border: 0;
     color: #9b9b9b;
-    font-size: 30px;
-    font-weight: 700;
-    height: 80px;
+    flex-shrink: 1;
+    font-size: 0.875rem;
```

Comment:
I feel the font size is very small now. Would like to deviate from the design and keep the default size.